### PR TITLE
Added wrapper for the GMM module

### PR DESCRIPTION
**Pull request by AlexisMignon**

### CODE REVIEW

**Comment by patricksnape** [comment]
jenkins test this please

File: cyvlfeat/gmm/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from .gmm import gmm
```
Comment (patricksnape) [inline cyvlfeat/gmm/__init__.py]: Missing new line

File: cyvlfeat/_vl/gmm.pxd [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Author: Alexis Mignon <alexis.mignon@probayes.com>
+# Date: Wed Jun  1 14:58:43 2016
+# Copyright (c) 2016 ProbaYes SAS
```
Comment (patricksnape) [inline cyvlfeat/_vl/gmm.pxd]: I'm not sure I want to start maintaining copyright notices at the top of each file. Would you be happy if we added an AUTHORS.md in the root that listed all contributors/joint copyright holders. Thus ensuring your contributions fall under the BSD license as well?

Comment (AlexisMignon) [inline cyvlfeat/_vl/gmm.pxd]: Sorry this just comes from the default settings of my editor

**Pull request merged**